src/main/java/com/brightcove/zencoder/client/request/ZencoderOutput.java

```diff
@@ -977,12 +987,17 @@ public void setHlsOptimizedTs(Boolean hls_optimized_ts) {
         this.hls_optimized_ts = hls_optimized_ts;
     }
 
-    public SegmentingType getPrepareForSegmenting() {
+    public List<SegmentingType> getPrepareForSegmenting() {
         return prepare_for_segmenting;
     }
 
     public void setPrepareForSegmenting(SegmentingType prepare_for_segmenting) {
```

Comment:
Now that `prepare_for_segmenting` is a `List`, a `set` method would not seem necessary, but in order to maintain backward compatibility, leaving this in there, and using it to reset the `List` to a single entry.